LISA/LISA.cpp

```diff
@@ -50,15 +50,14 @@ namespace Plugin {
 
             std::string path = (config.DbPath.IsSet() && !config.DbPath.Value().empty())
                             ? config.DbPath.Value() : service->PersistentPath();
-            if (_lisa->Configure(path)) {
-                TRACE(Trace::Information, (_T("LISA::Initialize register notification")));
-                _lisa->Register(&_notification);
-
-                TRACE(Trace::Information, (_T("LISA::Initialize register JSON-RPC API")));
-                Register(*this, _lisa);
-            } else {
-                message = _T("LISA could not be instantiated - could not initialize database.");
-            }
+
+            _lisa->Configure(path);
```

Comment:
Shouldn't be if(_lisa->Configure(path) == ERROR_NONE)? It seems that cause was that ERROR_NONE == 0.

Reply:
The thing is I don't really know what happens then, LISA is partially registered and some calls crash as in this ticket. So for now let's ignore it and let interface functions fail. 
I will investigate that when implementing proper Configure function.